File: Zadanie1 github/Python/Zadanie1.py
a = float(input("Pierwsza liczba: "))
znak = input("Co robimy (+, -, *, /): ")
b = float(input("Druga liczba: "))


if znak == "+":
    print(a + b)
elif znak == "-":
    print(a - b)
elif znak == "*":
    print(a * b)
elif znak == "/":
    print(a / b)
else:
    print("Zły znak!")

File: Zadanie1 github/C#/Zadanie1.cs
using System;


class Program {
    static void Main() {
       
        Console.WriteLine("Podaj pierwszą liczbę:");
        double a = double.Parse(Console.ReadLine());

        Console.WriteLine("Co robimy? (+, -, *, /):");
        string znak = Console.ReadLine();

        Console.WriteLine("Podaj drugą liczbę:");
        double b = double.Parse(Console.ReadLine());

       
        if (znak == "+") {
            Console.WriteLine(a + b);
        }
        else if (znak == "-") {
            Console.WriteLine(a - b);
        }
        else if (znak == "*") {
            Console.WriteLine(a * b);
        }
        else if (znak == "/") {
            Console.WriteLine(a / b);
        }
    }
}